nomenclature/testing.py

```diff
@@ -28,9 +29,13 @@ def assert_valid_yaml(path: Path):
 
 
 def assert_valid_structure(
-    path: Path, mappings: str = "mappings", definitions: str = "definitions"
+    path: Path,
```

Comment:
The order of arguments is not consistent with the CLI `validate-project` - and I think it will be way more common to use the "dimensions" options than custom folder names, so please order as dimensions-first...